packages/mdc-select/mdc-select.scss

```diff
@@ -133,6 +133,9 @@
 // postcss-bem-linter: end
 
 .mdc-select__menu {
+  position: absolute;
+  z-index: 3;
```

Comment:
The menu already has a `z-index` of 4

Reply:
So it's normal that it goes above the header?

Reply:
Hmm, I'm not sure. Setting its `z-index` to 3 may make it go under some other components.

Reply:
So I'll leave value 4 in order to avoid some unnoticed changes of behaviour in other situations for now.